### PR TITLE
streetsCalculator,street implemented, generator modified

### PR DESCRIPTION
**Pull request by Mankur11**

### CODE REVIEW

File: classes/streetsCalculator.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+export default class StreetsCalculator extends Calculator {
+    constructor() {
+        super();
+        this.streets = Generator.generateStreets();
+    }
+
+    run() {
+        this.log();
+        for (let street of this.streets) {
```
Comment (ostapgv) [inline classes/streetsCalculator.js]: With forEach it will look like `this.streets.forEach(street => street.log());`

Comment (ostapgv) [reply classes/streetsCalculator.js]: The same is for the parksCalculator.run

File: classes/generator.js [hunk 2]
```diff
@@ -17,4 +19,20 @@ export default class Generator {
             numOfTrees: Math.floor(Math.random() * 1500)
         });
     }
-}
\ No newline at end of file
+
+    static generateStreets() {
+        return streetsNames.map((streetName) => {
```
Comment (ostapgv) [inline classes/generator.js]: Just a small tip, you can omit using the `return` statement here.
`static generateStreets = () => streetsNames.map(streetName => this._generateStreetByName(streetName))`

Comment (ostapgv) [reply classes/generator.js]: The same is for the `generateParks` method.

File: classes/streetsCalculator.js (hunk 1 above)
Comment (Mankur11) [reply classes/streetsCalculator.js]: replaced

File: classes/generator.js (hunk 2 above)
Comment (Mankur11) [reply classes/generator.js]: fixed

**Pull request merged**